lib/releases.js

```diff
@@ -43,8 +43,14 @@ const findReleases = async ({
 
   log({ context, message: `Found ${releases.length} releases` })
 
+  const headRefRegex = /^refs\/heads\//
   const filteredReleases = filterByCommitish
-    ? releases.filter((r) => targetCommitish.match(`/${r.target_commitish}$`))
+    ? releases.filter(
+        (r) =>
+          // `refs/heads/branch` and `branch` are the same thing in this context
+          targetCommitish.replace(headRefRegex, '') ===
```

Comment:
This replace can be done once instead of for reach release.

Reply:
Of course :)